projects/qemu/run_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eu
+# Copyright 2025 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+cd $SRC/qemu/build-oss-fuzz
+make -j"$(nproc)" check-unit
```

Comment:
Not sure if we want to have this initial version while we figure out how to make `make check` work. WDYT @DavidKorczynski, @AdamKorcz? If so, should I add a comment here?

Reply:
I would say go for the the tests that work and thus avoid `make check`. Looking at the above logs it looks like there are ASAN issues with qemu and `make check`.

Our options are: build a version without ASAN or avoid these tests. I say at this point in time we should avoid the tests and thus avoid `make check`.

I'm happy to go for subset of tests that work well if the problems of the failing tests seem bigger than adjusting the set up in a minor manner. I think that should work for our usecases atm. This one seem to require adjustments a bit on the larger side to make it work.